packages/notation/notation.spec

```diff
@@ -0,0 +1,92 @@
+%global goproject github.com/notaryproject
+%global gorepo notation
+%global goimport %{goproject}/%{gorepo}
+
+%global gover 1.3.2
+%global rpmver %{gover}
+%global gitrev 001cc919603c1dc16c6aad387c94b4209cb9c901
+
+%global _dwz_low_mem_die_limit 0
+
+Name: %{_cross_os}%{gorepo}
+Version: %{rpmver}
+Release: 1%{?dist}
+Summary:  A CLI tool to sign and verify artifacts.
+License: Apache-2.0
+URL: https://%{goimport}
+Source0: https://%{goimport}/archive/v%{gover}/%{gorepo}-v%{gover}.tar.gz
+Source1: bundled-%{gorepo}-v%{gover}.tar.gz
+Source2: notation-trust-policy-json
+Source3: notation-tmpfiles.conf
+
+BuildRequires: git
```

Comment:
You don't need git for this build, right? No `git` repos are initialized, and there aren't patches applied.